core/iwasm/interpreter/wasm_runtime.c

```diff
@@ -83,6 +83,56 @@ wasm_unload(WASMModule *module)
     wasm_loader_unload(module);
 }
 
+void
+wasm_resolve_symbols(WASMModule *module)
+{
+    for (uint32 idx = 0; idx < module->import_function_count; ++idx) {
```

Comment:
had better declare `uint32 idx` first, it is a C source file, we usually do that way

Reply:
Fixed.